crates/sui-graphql-rpc/schema.graphql

```diff
@@ -174,6 +174,8 @@ enum AddressTransactionBlockRelationship {
 	AFFECTED
 }
 
+union Authenticator = Owner
```

Comment:
`Owner` is a bit of a funky type, so I want to make sure it's being used for the right reasons here, instead of `Address`. `Owner`s are entity that satisfies two properties:

- Can be referred to by an address.
- Are capable of owning an object (capable of having an object transferred to them or owning in the dynamic field/child object sense).

Today, there are really three distinct kinds of owner:

- `Address` (representing an account address).
- `Object` (representing an object on-chain, accessible by its ID at the root of the store).
- A wrapped object (has no other representation other than `Owner` in GraphQL currently).

When the `ObjectOwner` is a `Parent` we use the `Owner` type to represent that parent because it could be wrapped or not. When the `ObjectOwner` is an `AddressOwner` we use the `Owner` type again, but for different reason: It could be an address, or an object (which could also be wrapped).

If the `Owner` in the context of `Authenticator` has to be an address that corresponds to a public key, then the right type to use is `Address`, here.